openapi.yaml

```diff
@@ -0,0 +1,404 @@
+swagger: "2.0"
+info:
+  description: "reader api service"
+  version: "0.0.1"
+  title: "Reader API"
+  contact:
+    email: "hello@gorillab.co"
```

Comment:
should provide more info before the email:
- name: Gorillab
- url: gorillab.co